src/context/delay.ts

```diff
@@ -2,13 +2,17 @@ import { ResponseTransformer } from '../response'
 
 export const MIN_SERVER_RESPONSE_TIME = 100
 export const MAX_SERVER_RESPONSE_TIME = 400
+export const SERVER_RESPONSE_TIME_NODE = 5
 
-const getRandomServerResponseTime = () =>
-  Math.floor(
+const getRandomServerResponseTime = () => {
+  if (typeof window === 'undefined') {
```

Comment:
Just a thought here - I noticed the conversation around using a different mechanism for determining whether the process was running inside node.

There's a library here that does this: https://github.com/iliakan/detect-node

Their mechanism is very simple though and it's just this:

```js
  
// Only Node.JS has a process variable that is of [[Class]] process
module.exports = Object.prototype.toString.call(typeof process !== 'undefined' ? process : 0) === '[object process]';
```

Perhaps you could either just pull this lib in, or just directly inline this function?

If you do inline this function, it might be worth extracting it as a function named `detectNode` or something equivalent, just because this might turn out to be useful for other functionality in the project.

Just a thought!

Reply:
Hey, @citypaul. Thanks for the suggestion! 
I think we can start by including that snippet directly as a utility function in the library. You're right, we may want to parametrize the behavior in other parts, depending on whether it's running in Node or not.